Manifests/ManagedPreferencesApplications/com.twocanoes.xcreds.plist

```diff
@@ -520,7 +520,71 @@ A profile can consist of payloads with different version numbers. For example, c
 			<key>pfm_name</key>
 			<string>passwordElementID</string>
 			<key>pfm_title</key>
-			<string>Password Element ID</string>
+			<string>passwordElementID</string>
+			<key>pfm_type</key>
+			<string>string</string>
+		</dict>
+		<dict>
+			<key>pfm_default</key>
+			<string>given_name</string>
+			<key>pfm_description</key>
+			<string>Local DS to OIDC Mapping for First Name</string>
+			<key>pfm_documentation_url</key>
+			<string>https://github.com/twocanoes/xcreds/wiki/AdminGuide#oidcmappingfirstname</string>
+			<key>pfm_name</key>
+			<string>map_firstname</string>
+			<key>pfm_note</key>
+			<string>Map firstName to OIDC claim</string>
+			<key>pfm_title</key>
+			<string>First Name OIDC Mapping</string>
+			<key>pfm_type</key>
+			<string>string</string>
+		</dict>
+		<dict>
+			<key>pfm_default</key>
+			<string>family_name</string>
+			<key>pfm_description</key>
+			<string>Local DS to OIDC Mapping for Last Name</string>
+			<key>pfm_documentation_url</key>
+			<string>https://github.com/twocanoes/xcreds/wiki/AdminGuide#oidcmappinglastname</string>
+			<key>pfm_name</key>
+			<string>map_lastname</string>
+			<key>pfm_note</key>
+			<string>Map lastName to OIDC claim</string>
+			<key>pfm_title</key>
+			<string>Last Name OIDC Mapping</string>
+			<key>pfm_type</key>
+			<string>string</string>
+		</dict>
+		<dict>
+			<key>pfm_default</key>
+			<string>name</string>
+			<key>pfm_description</key>
+			<string>Local DS to OIDC Mapping for Name</string>
+			<key>pfm_documentation_url</key>
+			<string>https://github.com/twocanoes/xcreds/wiki/AdminGuide#oidcmappingfullname</string>
+			<key>pfm_name</key>
+			<string>map_fullname</string>
+			<key>pfm_note</key>
+			<string>Map fullName to OIDC claim</string>
+			<key>pfm_title</key>
+			<string>Full Name OIDC Mapping</string>
+			<key>pfm_type</key>
+			<string>string</string>
+		</dict>
+		<dict>
+			<key>pfm_default</key>
+			<string>name</string>
+			<key>pfm_description</key>
+			<string>Local DS to OIDC Mapping for Name</string>
+			<key>pfm_documentation_url</key>
+			<string>https://github.com/twocanoes/xcreds/wiki/AdminGuide#oidcmappinglusername</string>
+			<key>pfm_name</key>
+			<string>map_username</string>
```

Comment:
One last. This issue affects all of your new keys so maybe I'm getting it wrong and if so I'm happy to be corrected.

Reply:
See https://github.com/DigiDNA/ProfileManifests/pull/2#discussion_r1117045104